src/components/EditorSearchBar.css

```diff
@@ -8,7 +8,7 @@
 
 .search-bar i {
   display: block;
-  padding: 13px 0 0 13px;
+  padding: 13px;
```

Comment:
there was a need for right/left padding, and I could do it with the postcss thing but it has a box sizing: border box so I figured why not have padding from both of them